src/models.ts

```diff
@@ -16,8 +16,11 @@ export interface TileModel {
   description?: string;
   collectionIdentifier?: string;
   collectionName?: string;
+  collections?: string[];
```

Comment:
There are two `collections` arrays in here... also on line 21.

Reply:
fixed